src/main/java/in/gov/cdc/cdcdemostage/validators/dob/policy/DoBDeclarationPolicy.java

```diff
@@ -0,0 +1,47 @@
+package in.gov.cdc.cdcdemostage.validators.dob.policy;
+
+import in.gov.cdc.cdcdemostage.models.ExtractedPacket;
+import in.gov.cdc.cdcdemostage.models.ValidationError;
+import in.gov.cdc.cdcdemostage.validators.specs.IPolicyValidator;
+import org.springframework.stereotype.Component;
+
+import java.util.BitSet;
+import java.util.Optional;
+
+/**
+ * This class handles the Dob declaration policy of UIDAI. It checks for DobType
+ * and based on that value of the date field and as well availaibility of the
+ * supporting documents
+ */
+@Component
+public class DoBDeclarationPolicy implements IPolicyValidator {
+    @Override
+    public boolean supports(BitSet b) {
+        return false;
+    }
+
+    @Override
+    public String mode(String mode) {
+        return "ONLINE";
```

Comment:
My understanding is that we are checking if this policy is applicable for online mode here. If that is the case, it is possible that same policy can be applicable for different modes. 
Also we are setting and returning the same thing. This might be incorrect

Reply:
This code requires change. Yet to address. Idea is to support ONLINE/OFFLINE/* - * means applicable for all kinds of requests.